FDW - Release branch (#3331)
EnterpriseDB/docs: product_docs/docs/hadoop_data_adapter/2/07_features_of_hdfs_fdw.mdx

## Join pushdown

> Hadoop Foreign Data Wrapper supports join pushdown. It pushes the joins between the foreign tables of the same remote Hive or Spark server to that remote Hive or Spark server, enhancing the performance. From version 2.3.0 and later, you can enable the join pushdown at session and query level using the GUC.

Reviewer: ```suggestion
From version 2.3.0 and later, you can enable the join pushdown at the session and query level using the GUC variable.
```
Can you list the variable name?

Author: changes done as per both comments

For more information, see [Example: Aggregate pushdown](10b_example_aggregate_pushdown).

## ORDER BY pushdown

Hadoop Foreign Data Wrapper supports order by push-down. If possible, push `ORDER BY` clause to the remote server so that we get the ordered result set from the foreign server itself. It might help us to have an efficient merge join. 

> - [Example: ORDER BY pushdown](10c_example_order_by_push_down)

Reviewer: You should avoid single item bulleted lists. Use the same format as the other sections in the topic: For more information, see [Example: Aggregate pushdown](https://deploy-preview-3331--edb-docs-staging.netlify.app/docs/hadoop_data_adapter/latest/10b_example_aggregate_pushdown/).

Author: changes done as per both comments

## LIMIT OFFSET pushdown

> Hadoop Foreign Data Wrapper supports limit offset push-down. Wherever possible, perform LIMIT and OFFSET operations on the remote server. This reduces network traffic between local PostgreSQL and remote hdfs/Hive servers. ALL/NULL options are not supported on the Hive server, and thus they are not pushed down. Also, OFFSET without LIMIT is not supported on the remote server hence queries having that construct are not pushed.

Reviewer: - Is this specific to PostgreSQL or should we be using Postgres here?
- Use consistent capitalization for Hive and hdfs. I suggest Hive and HDFS. 
- Use contractions: are not = aren't, is not = isn't

Author: changes done as per both comments
